prow/apis/prowjobs/v1/types.go

```diff
@@ -487,6 +487,16 @@ type DecorationConfig struct {
 	// set the same as this request. Could be overridden by memory request
 	// defined explicitly on prowjob.
 	DefaultMemoryRequest *resource.Quantity `json:"default_memory_request,omitempty"`
+
+	// PodPendingTimeout is after how long the controller will perform a garbage
+	// collection on pending pods. Defaults to 10 minutes.
```

Comment:
1. Are these defaults what we already have today? (If they're different, can you explain why?)
2. Can you add integration tests, if possible, for these new fields?

Reply:
1. according to this part, https://github.com/kubernetes/test-infra/blob/master/prow/config/prow-config-documented.yaml#L1087, we have three variables I considered them.
2. sure, I will do it shortly

Reply:
> 1. Are these defaults what we already have today? (If they're different, can you explain why?)
> 2. Can you add integration tests, if possible, for these new fields?

1. 1according to this part, https://github.com/kubernetes/test-infra/blob/master/prow/config/prow-config-documented.yaml#L1087, we have three variables I considered them.
2. sure, I will do it shortly